topic_folders/communications/guides/adhere-style-guide.md

```diff
@@ -6,6 +6,18 @@ For example, headings should use title casing, and tools like the [Title Case Co
 
 Another meaningful example is spellings. Because The Carpentries is a global community-led organisation, content standardisation is important. Our Style Guide currently recommends the use of British English in all our resources across the board. More about this in the language section of our Style Guide.
 
+## Formatting Hyperlinks
+
+Hyperlinks on the websites for The Carpentries and its lesson programs should be formatted using [Jekyll's link formatting syntax](https://jekyllrb.com/docs/liquid/tags/#links).  This includes: 
+
+- using `{% link %}` for pages. For example, `[Become a member organisation]({% link pages/membership.md%})`
+- using `{% post %}` for blog posts. For example `[blog post]({% post _posts/2021/05/2021-05-01-blogging-from-the-future.md %})`
```

Comment:
I haven't tried this, I always use the `{% link %}` command. I'm not sure if there is a difference between the 2 besides not having to specify the extension. But it's probably fine to leave it as is for the docs.